pkg/pgmodel/extension.go

```diff
@@ -289,5 +289,9 @@ func getNewExtensionVersion(extName string,
 
 // trimLeadingZeros removes the leading zeros passed in the version number.
 func trimLeadingZeros(s string) string {
-	return strings.TrimLeft(s, "0")
+	rv := strings.TrimLeft(s, "0")
+	if rv == "" {
+		rv = "0"
+	}
+	return rv
```

Comment:
```suggestion
	if s = strings.TrimLeft(s, "0"); s == "" {
		return "0"
	}
	return s
```

Let's re-use the same variable.